Cookbook/Style-guide/README.md

```diff
@@ -764,6 +762,8 @@ Only explicitly use `open`, `public`, and `internal` when you require a full acc
 
 Use access control as the leading property specifier. The only things that should come before access control are the `static` specifier or attributes such as `@IBAction`, `@IBOutlet` and `@discardableResult`.
 
+Use `private`, `fileprivate`, `internal`, and `public` visibility qualifiers on each function implemented in an extension, rather than have a single qualifier for the entire extension.
```

Comment:
Maybe we should be consistent in our vocabulary, and either use "access control" or "access control qualifiers" or "visibility qualifiers", but not a mix of both between rules (e.g. this one and the previous one). Not sure what's best

Reply:
```suggestion
Use `private`, `fileprivate`, `internal`, and `public` access-level modifiers on each function implemented in an extension, rather than having a single access-level modifier for the entire extension.
```

Swift doc uses the phrase "access-level modifier", so we can follow it:
https://docs.swift.org/swift-book/LanguageGuide/AccessControl.html